### PR TITLE
[cirque] Basic Cluster Atrributes test with border router

### PR DESCRIPTION
**Pull request by doublemis1**
* Refactor python chip controller test_scripts
 * Add Basic Cluster Attributes test on cirque

### CODE REVIEW

**Comment by doublemis1** [comment]
To verify Basic Cluster Attributes, PR-6761 (https://github.com/project-chip/connectedhomeip/pull/6761) need to be merged.

**Comment by doublemis1** [comment]
The OTBR docker:

If the or-br-posix will be updated to the newer version then the dockerfile would look as follow:
```
FROM connectedhomeip/chip-cirque-device-base:latest

COPY entrypoint.sh /app
ENTRYPOINT ["/app/entrypoint.sh"]
```
Version which is used (81a4392f7f757482f37822c5e9d7fa4a899b8184, March 2) is too old to get working SRP.

**Review by Damian-Nordic — APPROVED** [review]

File: integrations/docker/ci-only-images/chip-cirque-device-base/border_router/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+WORKDIR /app
+
+# TODO: Use multi stage build for smaller image size.
+
+RUN apt-get update \
+  && apt-get install --no-install-recommends -y sudo git ca-certificates psmisc dhcpcd5 wpasupplicant wireless-tools \
+  && apt-get install -y libglib2.0 avahi-daemon libavahi-client3 \
+  && ln -fs /usr/share/zoneinfo/UTC /etc/localtime \
+  && git clone https://github.com/openthread/ot-br-posix . \
```
[the remark this answers is not in the document]
Comment (doublemis1) [reply integrations/docker/ci-only-images/chip-cirque-device-base/border_router/Dockerfile]: Already changed to:
```
  && git init \
  && git remote add origin https://github.com/openthread/ot-br-posix \
  && git fetch --depth 1 origin $OT_BR_POSIX_CHECKOUT \
  && git checkout FETCH_HEAD \
  && git submodule update --init --depth=1 \
  ```

File: src/controller/python/test/test_scripts/basic-cluster-attributes-test.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                                   nodeid=1),
+              "Failed to finish key exchange")
+
+    FailIfNot(test.TestNetworkCommissioning(nodeid=1,
+                                            endpoint=ENDPOINT_ID,
+                                            group=GROUP_ID,
+                                            dataset=TEST_THREAD_NETWORK_DATASET_TLV,
+                                            network_id=TEST_THREAD_NETWORK_ID),
+              "Failed to finish network commissioning")
+    time.sleep(10)
```
[the remark this answers is not in the document]
Comment (doublemis1) [reply src/controller/python/test/test_scripts/basic-cluster-attributes-test.py]: You are right, I will remove it.

**Pull request closed without merging**